skiko/buildSrc/src/main/kotlin/properties.kt

```diff
@@ -43,13 +43,13 @@ fun compilerForTarget(os: OS, arch: Arch): String =
             Arch.Wasm -> "Unexpected combination: $os & $arch"
         }
         OS.Android -> "clang++"
-        OS.Windows -> "cl.exe"
+        OS.Windows -> "clang-cl.exe"
```

Comment:
It seems we still use Visual C++ compiler on Windows?

![Uploading image.png…]()

Reply:
I see, missed that